src/main/java/org/openstreetmap/atlas/geography/atlas/dynamic/DynamicAtlasExpander.java

```diff
@@ -93,7 +93,7 @@ public DynamicAtlasPolicy getPolicy()
 
     boolean areaCovered(final Area area)
     {
-        if (this.areaCoveredCache.contains(area.getIdentifier()))
+        if (!entityNotCached(area))
```

Comment:
Double negative --> 🤯

Reply:
I wanted to be able to use the shortcut here ;P https://github.com/osmlab/atlas/pull/571/files#diff-8b62a99e1caf05da028d568d14101c2eR453

Reply:
Agreed, that makes that part cleaner!